docs/next/guides/accessibility/keyboard-shortcuts.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Keyboard shortcuts
+metaTitle: Keyboard shortcuts - Guide - Handsontable Documentation
+permalink: /next/keyboard-shortcuts
+canonicalUrl: /keyboard-shortcuts
+tags:
+- key bindings
+- keymap
+- shortcut manager
+- keyboard navigation
+---
+
+# Keyboard shortcuts
+
+Use and manage Handsontable's keyboard shortcuts.
+
+[[toc]]
+
+## About keyboard shortcuts
+
+You can intuitively navigate Handsontable with a keyboard, using the out-of-the-box [default keyboard shortcuts](#default-keyboard-shortcuts).
+
+You can also customize the entire set of keyboard shortcuts and handle assigned actions, by:
+- [Removing default keyboard shortcuts](#removing-default-keyboard-shortcuts)
+- [Adding custom keyboard shortcuts](#adding-custom-keyboard-shortcuts)
+- [Replacing default keyboard shortcuts](#replacing-default-keyboard-shortcuts)
+- [Using beforeKeyDown hook](#using-beforekeydown-hook)
+
+## Default keyboard shortcuts
+
+By default, Handsontable features the following keyboard shortcuts:
+
+#### NAVIGATION
```

Comment:
@warpech - is there a reason why we don't want to use the `:::details` container here? IMO:
- This is a classic **reference** information type – it should be separated from other types (like conceptual info or detailed procedures)
- Without the accordion, the page gets quite long and messy

I'm guessing that the ease of using Ctrl+F is the priority here, so I'm leaving it as is for now. But I'm using `::: details` in other analogical situations (e.g. [here](https://handsontable.com/docs/building/#building-the-javascript-package)), so it's something to think about.